CODEOWNERS

```diff
@@ -47,3 +47,6 @@
 
 # Approvers for Traditional Chinese contents
 /content/zh-tw/ @pichuang @ydFu
+
+# Approvers for English content
+/content/en/ @nate-double-u 
```

Comment:
Could you place this section around line 14? Since English is (currently) the main source of original documents. 

Also, I guess we need to add other maintainers in this subset.

```
# Order is important; the last matching pattern takes the most
# precedence. When someone opens a pull request that only
# modifies JS files, only @js-owner and not the global
# owner(s) will be requested for a review.
*.js    @js-owner #This is an inline comment.
```

```suggestion
/content/en/ @caniszczyk @CathPag @jasonmorgan @seokho-son @iamNoah1 @jihoon-seo @nate-double-u 
```

(@CathPag - do we need to minimize -reviewers(code-owners)- for English specific content?)

Reply:
I've made the update.

Reply:
LGTM! :)